fastlane_core/lib/fastlane_core/ui/interface.rb

```diff
@@ -141,8 +143,8 @@ def crash!(exception)
     # Basically this should be used when you actively catch the error
     # and want to show a nice error message to the user
     def user_error!(error_message, options = {})
-      options = { show_github_issues: false }.merge(options)
-      raise FastlaneError.new(show_github_issues: options[:show_github_issues]), error_message.to_s
+      options = { show_github_issues: false, error_info: nil }.merge(options)
```

Comment:
You should be able to pass `options` straight through, as both of these are the same as the default values.